common/script/content.coffee

```diff
@@ -951,10 +952,75 @@ api.spells =
         if !target.items.special.valentineReceived
           target.items.special.valentineReceived = []
         target.items.special.valentineReceived.push user.profile.name
+        target.flags.cardReceived = true
 
         target.markModified? 'items.special.valentineReceived'
         user.stats.gp -= 10
 
+    greeting:
+      text: t('greetingCard')
+      mana: 0
+      value: 10
+      immediateUse: true
+      silent: true
+      target: 'user'
+      notes: t('greetingCardNotes')
+      cast: (user, target) ->
+        if user == target
+          user.achievements.greeting ?= 0
+          user.achievements.greeting++
+        else
+          _.each [user,target], (t)->
+            t.achievements.greeting ?= 0
+            t.achievements.greeting++
+        if !target.items.special.greetingReceived
+          target.items.special.greetingReceived = []
+        target.items.special.greetingReceived.push user.profile.name
+        target.flags.cardReceived = true
+
+        target.markModified? 'items.special.greetingReceived'
+        user.stats.gp -= 10
+
+    thankyou:
+      text: t('thankyouCard')
+      mana: 0
+      value: 10
+      immediateUse: true
+      silent: true
+      target: 'user'
+      notes: t('thankyouCardNotes')
+      cast: (user, target) ->
+        if user == target
+          user.achievements.thankyou ?= 0
+          user.achievements.thankyou++
+        else
+          _.each [user,target], (t)->
+            t.achievements.thankyou ?= 0
+            t.achievements.thankyou++
+        if !target.items.special.thankyouReceived
+          target.items.special.thankyouReceived = []
+        target.items.special.thankyouReceived.push user.profile.name
+        target.flags.cardReceived = true
+
+        target.markModified? 'items.special.thankyouReceived'
+        user.stats.gp -= 10        
+        
+api.cardTypes =
```

Comment:
Hmmm... my idea led me down a refactoring rabbit hole. Going to table it for now, but it's something I want to do in the near future.

The gist of it is to separate out the cards and special spells (spook dust, seafoam, etc) from the normal spells.